files/.github/workflows/push-dist.yml

```diff
@@ -0,0 +1,31 @@
+# Because this library needs to be built,
+# we can't easily point package.json files at the git repo for easy cross-repo testing.
+#
+# This workflow brings back that capability by placing the compiled assets on a "dist" branch
+# (configurable via the "branch" option below)
+name: Push dist
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  push-dist:
+    name: Push dist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3<% if (pnpm) {%>
+      - uses: NullVoxPopuli/action-setup-pnpm@v2<%} else {%>
```

Comment:
Why do we need the different setup for pnpm here? Is it worth the additional complexity here?